nxdrive/manager.py

```diff
@@ -549,6 +539,8 @@ def set_auto_update(self, value: bool, /) -> None:
     def generate_report(self, *, path: Path = None) -> Path:
         from .report import Report
 
+        log.info(f"Features: {Feature.__dict__}")
+        log.info(f"Options: {Options!r}")
```

Comment:
It is enough to do:
```suggestion
        log.info(f"Features: {Feature}")
        log.info(f"Options: {Options}")
```
- `Features` is good as-is.
- `str(Options)` will show only modified options, which is what is prefered.